mlbgame/data.py

```diff
@@ -1,10 +1,22 @@
+#!/usr/bin/env python
+
 '''
 This module gets the XML data that other functions use.
 It checks if the data is cached first, and if not, 
 gets the data from mlb.com
 '''
+
+from __future__ import print_function
+
+import sys
 import os
-import urllib2 as url
+
+if sys.version_info[0] == 2:
+    from urllib2 import urlopen
+else:
+    from urllib.request import urlopen
```

Comment:
@crwilcox Yes I also agree with what you are saying, I have seen the try/except method used much more often than an if statement.